ci-operator/config/opendatahub-io/kubeflow/opendatahub-io-kubeflow-main.yaml

```diff
@@ -83,16 +75,7 @@ tests:
       SOURCE_IMAGE_REF: kf-notebook-controller
     env:
       IMAGE_REPO: kubeflow-notebook-controller
-      RELEASE_VERSION: master
-    workflow: opendatahub-io-ci-image-mirror
-- as: kf-notebook-controller-image-mirror-latest
-  cron: 0 2 * * *
```

Comment:
With this, no jobs are croned anymore here - is this desired? No idea, just asking :thinking:

Reply:
Yes, we don't use chron-jobs on the config anymore, I guess this was for some old procedure that we had in ODH